html/janus.js

```diff
@@ -1545,6 +1545,10 @@ function Janus(gatewayCallbacks) {
 			// TODO
 		};
 		if(!incoming) {
+			if(!config.pc) {
+				Janus.warn("Invalid PeerConnection");
+				return true;
+			}
```

Comment:
I think this whole check should be moved at the beginning of the method, where `config` is assigned. It the PeerConnection doesn't exist, it doesn't make sense to proceed with the method either way. As a side note, there's no return value for the method, so just `return;` is enough.

Reply:
Sure. Done